src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/SuperCloneCheck.java

```diff
@@ -37,7 +37,20 @@
  * <pre>
  * &lt;module name=&quot;SuperClone&quot;/&gt;
  * </pre>
+ * <p>Example: </p>
+ * <pre>
+ * public class Foo {
+ *
+ * public Object clone() {return null;} // violation, does not call super.clone()
```

Comment:
It is not valid java code to have all these clones in 1 single method. Also the examples are not very creative in that they all look very similar.
What about an example where super clone is not called in the return. Like it is assigned to a variable, with some more modifications and then returned.
`clone` can't have any parameters and can't be native.

Look at the source code of the check to get an idea of some other examples that users may not think of beyond the basics.https://github.com/checkstyle/checkstyle/blob/bd45909e6345b2d3ef5c5472d35a3ad926a31eb9/src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/AbstractSuperCheck.java